zubhub_backend/zubhub/activitylog/serializers.py

```diff
@@ -0,0 +1,35 @@
+from creators.models import Creator
+from rest_framework import serializers
+
+from .models import Activitylog
+from creators.serializers import CreatorMinimalSerializer
+
+class CreatorInfoSerializer(serializers.ModelSerializer):
+    tags = serializers.SlugRelatedField(slug_field="name",
+                                        read_only=True,
+                                        many=True)
+    # badges = serializers.SlugRelatedField(slug_field="badge_title",
+    #                                     read_only=True,
+    #                                     many=True)
+
+    class Meta:
+        model = Creator
+        fields = ('id', 'username', 'followers_count', 
+                  'following_count', 'projects_count', "tags", 
+                #   "badges"
+                  )
+
+
+class ActivitylogSerializer(serializers.ModelSerializer):
+
+    source = CreatorInfoSerializer()
+    target = CreatorInfoSerializer()
+
+    class Meta:
+        model = Activitylog
+        fields = ('id', 'source', 'target', 'date', 'type', 'link', 'message')
+        # depth  = 1
```

Comment:
Same goes for this field. Consider removing it as well. If it is here for some reason, may be add a comment.

Reply:
Done.